group_vars/nfss.yml.sample

```diff
@@ -56,6 +56,11 @@ dummy:
 # addresses. (Note: ganesha ignores this parameter in current version due to
 # this bug: https://github.com/nfs-ganesha/nfs-ganesha/issues/217)
 # ceph_nfs_bind_addr: 0.0.0.0
+# Minimize or turn off ganesha's caching such as dentry and fd caching.
+# Currently, the caching is turned on by default.
+# When using ganesha as CephFS's gateway, it's recommended to turn off
+# ganesha's caching as the libcephfs clients also cache the same information.
+# ceph_nfs_caching_off: false
```

Comment:
The sample is generated from `roles/ceph-nfs/defaults/main.yml`. You don't need to change this.

Reply:
will do

Reply:
AS @batrick suggested you need to edit `roles/ceph-nfs/defaults/main.yml` only and then run `generate_group_vars_sample.sh`.